internal/service/health.go

```diff
@@ -18,6 +20,14 @@ func NewHealthService(backendUsecase *biz.IsBackendAvaliableUsecase) *HealthServ
 	}
 }
 
+func (s *HealthService) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
+	log.Infof("AuthFuncOverride called for method: %s", fullMethodName)
```

Comment:
Might make more sense to be `Debugf` instead not sure how commonly this would be fired

Reply:
I need to remove this method. Selector middlware doesn't work in case of stream grpc, i was experimenting with adding some skip functionality to over ride the middleware auth